android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -223,14 +238,47 @@ public class CustomTwilioVideoView extends View
     // Map used to map remote data tracks to remote participants
     private final Map<RemoteDataTrack, RemoteParticipant> dataTrackRemoteParticipantMap = new HashMap<>();
 
+    private final ActivityEventListener activityEventListener = new BaseActivityEventListener() {
+        @Override
+        public void onActivityResult(Activity activity, int requestCode, int resultCode, Intent data) {
+            super.onActivityResult(activity, requestCode, resultCode, data);
+            if (requestCode == REQUEST_MEDIA_PROJECTION) {
+                if (resultCode != Activity.RESULT_OK) {
+                } else {
+                    screenCapturer = new ScreenCapturer(themedReactContext, resultCode, data, new ScreenCapturer.Listener() {
+                        @Override
+                        public void onFirstFrameAvailable() {
+                        }
+
+                        @Override
+                        public void onScreenCaptureError(String errorDescription) {
+                            if (!isStoppingScreenShare) {
+                                stopScreenCapture();
+                            }
+                        }
+                    });
+                    startScreenCapture();
+                }
+            }
+        }
+    };
+
     public CustomTwilioVideoView(ThemedReactContext context) {
         super(context);
         this.themedReactContext = context;
         this.eventEmitter = themedReactContext.getJSModule(RCTEventEmitter.class);
 
         // add lifecycle for onResume and on onPause
         themedReactContext.addLifecycleEventListener(this);
+        Activity currentActivity = context.getCurrentActivity();
```

Comment:
Newline and comment for this would be nice:
```suggestion

         // Create activity for media projection (screen sharing) permission events
        Activity currentActivity = context.getCurrentActivity();
```